rpc/wrtc_base_channel.go

```diff
@@ -83,8 +92,6 @@ func newBaseChannel(
 				return
 			}
 
-			ch.mu.Lock()
-			defer ch.mu.Unlock()
 			switch connectionState {
 			case webrtc.ICEConnectionStateDisconnected,
 				webrtc.ICEConnectionStateFailed,
```

Comment:
Can't comment on the line, but I think the `peerConn.GracefulClose` on L120 is now redundant.

Reply:
Removed